lib/stack_master/commands/tidy.rb

```diff
@@ -0,0 +1,68 @@
+module StackMaster
+  module Commands
+    class Tidy
+      include Command
+      include StackMaster::Commands::TerminalHelper
+
+      def initialize(config)
+        @config = config
+      end
+
+      def perform
+        used_templates = []
+        used_parameter_files = []
+
+        templates = Set.new(find_templates())
+        parameter_files = Set.new(find_parameter_files())
+
+        status = @config.stacks.each do |stack_definition|
```

Comment:
I feel like this could be a separate method like `def remove_used(templates, parameter_files)` for clarity. (or maybe even one for each?) But that's just nitpicking :)

Reply:
Had a go at this but as it's referencing multiple sets (and has side effects where it prints missing templates) it complicates things a bit. Decided it's not worth it for this length of method.